src/utils.ts

```diff
@@ -200,7 +200,7 @@ export function getArgs () {
       description: 'Select the suite to run'
     })
     .demandOption(['runCfgPath', 'suiteName'])
-    .argv;
+    .parseSync();
```

Comment:
Mostly a typescript thing that's affecting us due to a dep upgrade. `argv` now returns _either_ a normal object _or_ a promise, if you have any async commands. Because of this, we'd have to `await` on the next line and turn this entire function into async and so on and so forth. By using `parseSync()`, we ensure that we always get back a plain old javascript object, just like we used to.